test.sh

```diff
@@ -127,8 +129,21 @@ make_docker_images () {
     done
 }
 
+# Pylint tests
+echo "PYLINT TESTS"
+
+if [ ! -n "${TEST_INTEGRATION+ }" ] && [ ! -n "${TEST_UNIT+ }" ]; then
+    $PYTHON -m pylint --disable=all --enable=E --enable=W --additional-builtins=_ *.py atomic Atomic tests/unit/*.py -d=no-absolute-import,print-statement,no-absolute-import,bad-builtin
+    $PYTHON3 -m pylint --disable=all --enable=E --enable=W --additional-builtins=_ *.py atomic Atomic tests/unit/*.py -d=no-absolute-import,print-statement,no-absolute-import,bad-builtin
+
+else
+    echo "      SKIPPING PYLINT TESTS..."
+fi
+
 make_docker_images
```

Comment:
Can we move this below unittests? My env requires sudo to run this function and it takes 8 seconds to attempt the build.